pykwasm/src/pykwasm/kdist/wasm-semantics/wasm.md

```diff
@@ -1187,7 +1184,7 @@ The `#take` function will return the parameter stack in the reversed order, then
                ...
          </instrs>
          <valstack>  VALSTACK => .ValStack </valstack>
-         <locals> LOCAL => .Map </locals>
+         <locals> LOCAL </locals>
```

Comment:
Why not use `Locals => .List`?

Reply:
Because this rule is immediately followed by the one for `init_locals` which rewrites this cell to the empty list.

Reply:
Makes sense! Thanks!